### PR TITLE
Disable blunderbuss in gitops like all other repos

### PR DESCRIPTION
**Pull request by mads-hartmann**
## Description
Blunderbuss doesn't work as expected and we have disabled it in all other repos. This also disabled it for this repo.

## Related Issue(s)

No issue, ad-hoc fix.

## How to test

N/A

## Release Notes

```release-note
NONE
```

## Documentation

N/A

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mads-hartmann** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by wulfthimm — APPROVED** [review]

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/60#pullrequestreview-826167438" title="Approved">wulfthimm</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)~~ [wulfthimm]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 2ac44e6024135b33472caacea2af23e96580ff66</details>

**Comment by roboquat** [comment]
@mads-hartmann: Updated the `plugins` configmap in namespace `prow` at cluster `default` using the following files:
 - key `plugins.yaml` using file `config/plugins.yaml`

<details>

In response to [this](https://github.com/gitpod-io/gitbot/pull/60):

>## Description
>Blunderbuss doesn't work as expected and we have disabled it in all other repos. This also disabled it for this repo.
>
>## Related Issue(s)
>
>No issue, ad-hoc fix.
>
>## How to test
>
>N/A
>
>## Release Notes
>
>```release-note
>NONE
>```
>
>## Documentation
>
>N/A


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
